### ISSUE TITLE
DM-VIO comparison

### ISSUE THREAD

**Issue by giangdao1402** [sair-lab/AirSLAM#179]
Hi @xukuanHIT 
Thank for your great work !
Have you tried to compare AirSLAM with DM-VIO [2022] ?

**Comment by xukuanHIT** [comment]
No, the selected baselines have been listed in the experimental section of the paper.
